central/cve/converter/convert_utils.go

```diff
@@ -261,6 +263,11 @@ func ProtoCVEToEmbeddedCVE(protoCVE *storage.CVE) *storage.EmbeddedVulnerability
 		SuppressActivation:    protoCVE.GetSuppressActivation(),
 		SuppressExpiry:        protoCVE.GetSuppressExpiry(),
 	}
+	if features.PostgresDatastore.Enabled() {
+		embeddedCVE.Cve = protoCVE.GetCve()
+	} else {
+		embeddedCVE.Cve = protoCVE.GetId()
```

Comment:
if CVE and ID are always the same, I think they are. Can we just change this to the top case?

Reply:
no..if postgres is enabled cve id is cve+os. Again, this is one of those cases where data is queried by ID.